src/main/plots/scoting_solutions_plots.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+
+from typing import List, Union, Tuple
+
+import pandas as pd
+
+from src.main.util.consts import ISO_ENCODING, CODE_TRACKER_COLUMN
+from src.main.util.file_util import get_parent_folder, get_name_from_path
+from src.main.preprocessing.code_anonimization import anonymize_code_in_df
+from src.main.plots.util.graph_representation_util import get_color_by_rate, get_graph_representation, create_dot_graph
+
+
+def __find_next_score_index(scores: List[float], start_index: int = 0) -> int:
+    for i in range(start_index, len(scores)):
+        if scores[i] != scores[start_index]:
+            return i
+    return -1
+
+
+def __get_label_for_score(id: int, label: Union[str, int], score: float) -> str:
+    return f'{id} [label="{label}", style=filled, fillcolor={get_color_by_rate(score)}]\n'
+
+
+def __get_edge(src_id: int, dst_id: int) -> str:
+    return f'{src_id} -> {dst_id}\n'
+
+
+def get_labels_and_graph_structure(scores: List[float]) -> Tuple[str, str]:
```

Comment:
Cannot find out what's wrong but I tested it on the simple list of scores [0, 0.1, 0.2, 0.3, 0.4, 0.4, 1] and got this (seems weird):
![image](https://user-images.githubusercontent.com/24303332/91236654-35d0fa80-e741-11ea-8099-34b3e7a66ba9.png)

Reply:
Sorry, I have fixed it